doc/classes/KinematicBody2D.xml

```diff
@@ -103,7 +103,8 @@
 			</argument>
 			<description>
 				Moves the body along a vector. If the body collides with another, it will slide along the other body rather than stop immediately. If the other body is a [KinematicBody2D] or [RigidBody2D], it will also be affected by the motion of the other body. You can use this to make moving or rotating platforms, or to make nodes push other nodes.
-				[code]linear_velocity[/code] is the velocity vector in pixels per second. Unlike in [method move_and_collide], you should [i]not[/i] multiply it by [code]delta[/code] — the physics engine handles applying the velocity.
+				This method should be used in [method _physics_process].
```

Comment:
Same as above:

```suggestion
				This method should be used in [method _physics_process] (or a method called by [method _physics_process]), as it uses the physics step's `delta` value automatically. Otherwise, the simulation won't run at the correct speed.
```

Reply:
I'll make it more clear. Also correct the method reference.